enlighten/scope/GainDBFeature.py

```diff
@@ -1,11 +1,11 @@
-from enlighten import util
+from .. import util
 import logging
+import enlighten
```

Comment:
What does this line add?

Reply:
It enables static tracing using the annotation on ln43.

Reply:
Ah...okay